### PR TITLE
added: chefOfTheWeekRouter, popualrResRouter, signatureDishRouter

### PR DESCRIPTION
**Pull request by shirrotem**

### CODE REVIEW

File: src/controllers/popularResController.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+import { RestaurantModel } from "../models/restaurant";
+
+export const getPopularRestaurants = async (
+  req: express.Request,
+  res: express.Response
+) => {
+  try {
+    const popularRestaurants = await RestaurantModel.find({
+      popularRestaurant: true,
+    });
```
Comment (MoranOshiaMoveo) [inline src/controllers/popularResController.ts:11]: find is giving only the first element, u'll need to use the filter on the array

File: src/controllers/signatureDishControllers.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+import { DishModel } from "../models/dish";
+
+export const getSignatureDish = async (
+  req: express.Request,
+  res: express.Response
+) => {
+  try {
+    const signatureDish = await DishModel.find({
+      signatureDish: true,
+    });
```
Comment (MoranOshiaMoveo) [inline src/controllers/signatureDishControllers.ts:11]: same in here

**Review by MoranOshiaMoveo — CHANGES_REQUESTED** [review]

File: src/controllers/popularResController.ts (hunk 1 above)
Comment (shirrotem) [reply src/controllers/popularResController.ts:11]: fixed

File: src/controllers/signatureDishControllers.ts (hunk 2 above)
Comment (shirrotem) [reply src/controllers/signatureDishControllers.ts:11]: fixed

**Pull request merged**